# Long-read sequencing reveals a pan-African structural variation landscape driving phenotypic diversity and local adaptation 
Africa harbors the greatest genetic and phenotypic diversity among humans, yet African populations remain significantly underrepresented in genomic research, particularly with respect to SVs. To address this gap, we constructed a high-quality pan-African SV reference  by integrating deep long-read sequencing of a Central African rainforest hunter-gatherer (RHG) individual with 30 publicly available African long-read genomes. We identified 25,714 SVs in the RHG genome, including 787 variants absent from existing databases, significantly enriched in regulatory regions linked to immune function and growth-related pathways. The resulting catalog comprises 87,288 SVs, of which 38,818 high-confidence variants were genotyped across populations, with individuals carrying an average of 12,404 SVs. SV-based phylogenetic and population structure analyses recapitulated major African demographic patterns, revealing deep divergence between San and RHG lineages as well as extensive gene flow associated with pastoralist and agricultural expansions. Integrating selection scans with regulatory annotations and CRISPR-Cas9 validation, we identified adaptive SVs affecting genes involved in skeletal biology (TNFRSF11B, FGFRL1), pigmentation (EIF3B), cardiometabolic traits (FABP2, PIK3C2A, RGS2), and immune responses (GSDMD, IL4I1, EXOC2, VNN2), demonstrating their role in local adaptation to distinct ecological and cultural environments. Our findings reveal SV as a fundamental driver of African phenotypic diversity and offers a new target for understanding health disparities in African populations.
